packages/gafl-webapp-service/src/pages/payment/cancelled/payment-cancelled.njk

```diff
@@ -4,7 +4,7 @@
 {% from "button/macro.njk" import govukButton %}
 
 {% set title = mssgs.payment_cancelled_title %}
-{% block pageTitle %}{{ title }}{{mssgs.gov_uk}}{% endblock %}
+{% block pageTitle %}{{ title }}{{ mssgs.header_service_name_title }}{{mssgs.gov_uk}}{% endblock %}
```

Comment:
Why don't you add in 'Error' here?

Reply:
There are no errors possible on this page, so i just updated the title to the correct format only

Reply:
It really shouldn't be in this ticket, I will fix this